src/components/Fieldset/__snapshots__/Fieldset.test.ts.snap

```diff
@@ -178,7 +178,7 @@ exports[`<Fieldset /> CheckboxField story renders snapshot 1`] = `
   <div
     aria-live="assertive"
     class="field-note checkbox-field__note"
-    id="mock-id"
+    id="6"
```

Comment:
matches `aria-describedby` on the legend above 👍  and `input / label` associations look right 👍